test/functional/wallet_listreceivedby.py

```diff
@@ -18,16 +18,17 @@ class ReceivedByTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 2
 
-    def import_deterministic_coinbase_privkeys(self):
-        assert_equal(0, len(self.nodes[1].listreceivedbyaddress(minconf=0, include_empty=True, include_watchonly=True)))
-        super().import_deterministic_coinbase_privkeys()
-        self.num_cb_reward_addresses = len(self.nodes[1].listreceivedbyaddress(minconf=0, include_empty=True, include_watchonly=True))
+    #def skip_test_if_missing_module(self):
+    #    self.skip_if_no_wallet()
```

Comment:
14631: ?

Reply:
These two lines were triggering deadcode test failure, I had to comment it to make CI happy

Reply:
just don't add these lines :) Upstream didn't add them in this PR